src/cljs/zetawar/subs.cljs

```diff
@@ -456,8 +455,9 @@
 
 (deftrack compatible-armor-types-for-repair? [conn targeted-q targeted-r]
   (when-let [[selected-q selected-r] @(selected-hex conn)]
-    (game/compatible-armor-types-for-repair? @conn @(game conn) @(unit-at conn selected-q selected-r)
-                                                                @(unit-at conn targeted-q targeted-r))))
+    (game/compatible-armor-types-for-repair? @conn @(game conn)
```

Comment:
What's the reason for this change? It should work either way as long as the needed properties are pulled by the unit-at sub, right?